tests/ts/fadvise/drop

```diff
@@ -18,33 +19,42 @@ FILE="ddtest"
 BS=4k
 COUNT=8
 
+FILE_FS="$("$TS_CMD_FINDMNT" -nr -o FSTYPE -T "$PWD")"
+if [[ "$FILE_FS" = "tmpfs" ]]; then
+	ts_skip "fincore does not work on tmpfs"
+fi
+
+create_file() {
+    dd if=/dev/zero of="$FILE" bs=$BS count=$COUNT conv=fsync >& /dev/null
+}
+
```

Comment:
Maybe it would be better to have the $FILE as an argument for the function rather than a global variable for readability, but it's not so important ;-)